docs/en/observability/synthetics-quickstart.asciidoc

```diff
@@ -57,15 +57,21 @@ heartbeat.monitors:
   type: browser
   schedule: "@every 1m"
   source:
-    local: <3>
-      path: "/opt/todos" <4>
+    zip_url: <3>
+      url: "https://github.com/elastic/synthetics/archive/refs/heads/master.zip" <4>
+      folder: "examples/todos" <5>
+      username: "" <6>
+      password: "" 
 ----
 <1> Each `monitor` gets its own ID in the {uptime-app} and, therefore its own schedule entry.
 This allows tests to be run in parallel and analyzed separately.
 <2> In this example, a synthetic test is defined inline. This is a two-step script that first loads
 a homepage and then hovers over a product menu. See <<synthetics-syntax>> for more information.
-<3> In this example, our library of synthetic tests come from https://github.com/elastic/synthetics/tree/master/examples/todos[our todos example]. The path shown here corresponds to the path {heartbeat} will look at inside the docker container.
-<4> Note that this path is local to the docker container, not the host system. You can have the todos files anywhere you like on the host, and map them to `/opt/todos` in the container.
+<3> In this example, our library of synthetic tests is downloaded from the
```

Comment:
Super nit and subjective:

```suggestion
<3> In this example, we download our library of synthetic tests from the
```

The tech writers have usually recommended that I avoid passive language, so this simplification is worth considering. We could also write `"This example downloads..."`

Reply:
That's a great point Justin however I think this is a different situation because we don't download the library it is downloaded by heartbeat at some later point on a schedule. At least that's how it reads to me. @dedemorton WDYT?